Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -377,31 +436,35 @@ function Enable-StorageEncryptionInTransit
         {
             if (-not $DryRun)
             {  
-                Write-Host  $([Constants]::DoubleDashLine)
-                Write-Host "[step 5 of 6] : Exporting the Storage Account to csv file so that it can be used to rollback."
-                Write-Host "Backing up config of Storage Account(s) details. Please do not delete this file. Without this file you won't be able to rollback any changes done through remediation script." 
-                $stgWithDisableHTTPS | Export-CSV -Path "$($folderpath)\StorageWithDisableHTTPS.csv" -NoTypeInformation
-                Write-Host "Path: $($folderPath)\StorageWithDisableHTTPS.csv" -ForegroundColor $([Constants]::MessageType.Update)
-                Write-Host $([Constants]::SingleDashLine)
-               
-                if (-not $Force)
+                if (-not $UseUserAssignedManagedIdentity -and -not $UseSystemAssignedManagedIdentity)
```

Comment:
Plz add comment for usage of this condition.

Reply:
Done.